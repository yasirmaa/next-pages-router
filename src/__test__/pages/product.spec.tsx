import ProductPage from '@/pages/product';
import { render } from '@testing-library/react';

describe('Product Page', () => {
  it('render product page', () => {
    const { container } = render(<ProductPage />);
    expect(container).toMatchSnapshot();
  });
});
