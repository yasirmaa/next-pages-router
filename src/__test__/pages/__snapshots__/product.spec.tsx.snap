// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Page render product page 1`] = `
<div>
  <div
    class="w-full px-20"
  >
    <h1
      class="className text-3xl font-semibold"
    >
      Product Page
    </h1>
    <div
      class="product-grid__items grid grid-cols-4 gap-5"
    />
  </div>
</div>
`;
